client/homebrew/brewRenderer/toolBar/toolBar.jsx

```diff
@@ -214,8 +248,19 @@ const ToolBar = ({ displayOptions, currentPage, totalPages, onDisplayOptionsChan
 					className='tool'
 					type='button'
 					title='Next Page(s)'
-					onClick={()=>scrollToPage(pageNum + 1)}
-					disabled={pageNum >= totalPages}
+					onClick={()=>{
+						// if there are multiple pages in a 'row' and they are in 'view',
+						// then the 'max'/last page in view will always be the same, and
+						// the other pages will always be the same (since the viewport doesn't change).
+						// So this needs to scroll to the 'max', then see what is newly in view,
+						// and if the same pages are visible, do it again but +1.
+						const start = _.max(visiblePages);
+						scrollToPage(start);
+						if(start === _.max(visiblePages)){
+							scrollToPage(start + 1);
+						};
+					}}
```

Comment:
I'm not sure I understand this process. Don't we always want to jump to `_max(visiblePages) + 1` from the get-go?

Reply:
Intuitively, I think you are right.  But i just remember fussing with this so much and I'm not sure why, and why I left such a big comment on something seemingly pretty simple.  It could just be a remnant scrap from earlier iteration that didn't get cleaned up....or, it portends doom to anyone that might involve themselves with deeply cursed code.  

As I said, you are probably right, but I need to come back with more time and retry it on my local before saying "yes/no".

Reply:
Going to just simplify this. I don't see any behavior difference.